composer.json

```diff
@@ -87,7 +87,7 @@
     "elasticsearch/elasticsearch": "^6.0.1",
     "friendsofphp/php-cs-fixer": "^2.14.0",
     "fp/jsformvalidator-bundle": "^1.5.1",
-    "fzaninotto/faker": "^1.7.1",
+    "fzaninotto/faker": "1.7.1",
```

Comment:
Unit Tests results should not be dependent on faker generating the data in my guess. And even so, this will still not solve tests like this: 

```
        $loginPage->login('no-reply@shopsys.com', 'user123');
        $me->see('Jaromír Jágr');
```